src/routes/docs/advanced/platform/+layout.svelte

```diff
@@ -101,6 +101,11 @@
                     new: true,
                     label: 'Phone OTP',
                     href: '/docs/advanced/platform/phone-otp'
+                },
+                {
+                    new: true,
```

Comment:
See [here](https://github.com/appwrite/website/blob/0900b4f39439ca28a3d808f9c2700e08cfad7e84/src/routes/docs/advanced/platform/%2Blayout.svelte#L102) on how we should mark these as not new with an expiration, 30 days in the future should suffice considering pr reviews and release.